specification/indices/_types/DataStream.ts

```diff
@@ -47,6 +53,11 @@ export class DataStream {
    *  If `true`, the data stream allows custom routing on write request.
    */
   allow_custom_routing?: boolean
+  /**
+   * Information about failure store backing indices
```

Comment:
I would appreciate a better description here - I usually use the docs for this but there aren't any for now.